activerecord/lib/active_record/delegated_type.rb

```diff
@@ -169,13 +169,14 @@ module DelegatedType
     #   entry.access_notice_message?
     def delegated_type(role, types:, **options)
       belongs_to role, options.delete(:scope), **options.merge(polymorphic: true)
-      define_delegated_type_methods role, types: types
+      define_delegated_type_methods role, types: types, options: options
```

Comment:
Can you update the documentation to reflect those new options?

Reply:
I updated it.